android/app/src/main/java/com/shapeshift/MainActivity.java

```diff
@@ -59,6 +60,7 @@ protected ReactActivityDelegate createReactActivityDelegate() {
   @Override
   protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
+    WebView.setWebContentsDebuggingEnabled(true);
```

Comment:
Just a thought, is there a possibility we want this disabled in production? Only reason I could imagine is performance reasons since we're open source and dont need to rely on weak obfuscation via minification or similar

Reply:
Actually an interesting one.
Technically that's really allowing debugger capabilities for the webview which you'd only see if you're running the mobile app from source with metro. As far as the sourcemaps are concerned, those are the ones of the webview i.e if prod, prod sourcemaps (i.e obsfucated), if develop, develop ones i.e proper ones same as localhost. 

Now while this is not inherently dangerous (nothing different than being able to access sourcemaps and run a debugger on prod, however ugly those sourcemaps may be), there is definitely a very small but non-negligeable risk and if we can disable it on prod, might as well

Reply:
c02c20e and 8f530ac570e5217ecdea53e392d595d3fdb5d1d7
Confirmed debugger is still happy in debug builds 
<img width="1726" alt="image" src="https://github.com/shapeshift/mobile-app/assets/17035424/38b8de5d-1352-4db1-bacc-37c618849b56">